### PR TITLE
Bug fixes for the nltk hook

### PR DESCRIPTION
**Pull request by motasay**
This should fix the issues introduced by #3705

### CODE REVIEW

**Comment by danielcooperxyz** [comment]
These changes fix the issues I have seen when compiling nltk data.

File: PyInstaller/hooks/hook-nltk.py [hunk 1]
```diff
@@ -17,7 +18,5 @@
 
 # loop through the data directories and add them
 for p in nltk.data.path:
-    datas.append((p, "nltk_data"))
-
-# nltk.chunk.named_entity should be included
-hiddenimports = ["nltk.chunk.named_entity"]
```
[the remark this answers is not in the document]
Comment (motasay) [reply PyInstaller/hooks/hook-nltk.py]: It should only be included if you need the `named_entity` module. Definitely not all projects need that so I removed it and a custom hook can be used to include it as needed.

**Comment by motasay** [comment]
Thanks @htgoebel I'll try to get back to this ASAP - sorry.

File: PyInstaller/hooks/hook-nltk.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (htgoebel) [reply PyInstaller/hooks/hook-nltk.py]: I just see that I mixed it up: `nltk.chunk.named_entity` was always included and now shall not. So this is fine.

> if you need the named_entity module.

I don't understand exactly. Do you mean: If `nltk.chunk.named_entity` is imported by the project to be frozen?

**Comment by Legorooj** [comment]
@motasay pinging

**Pull request closed without merging**